docs/sources/panels/visualizations/time-series/graph-time-series-stacking.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "Graph stacked time series"
+keywords = ["grafana", "time series panel", "documentation", "guide", "graph"]
+weight = 400
++++
+
+# Graph stacked time series
+
+> **Note:** This is a beta feature. Time series panel is going to replace the Graph panel in the future releases.
+
+This section explains how to use Time series field options to control the stacking of the series and illustrates what the stacking options do.
```

Comment:
```suggestion
This section explains how to use Time series panel field options to control the stacking of the series and illustrates what the stacking options do.
```

Reply:
Can you explain a bit more about what stacking is and why someone might want to do it?

Reply:
can i write a followup section about how you shouldn't do it? ;)

https://leeoniya.github.io/uPlot/demos/stacked-series.html

Reply:
If you like, sure. You could also write an educational topic about how to create graphs, what types of graphs go with what data, and that sort of thing. Our long-term goal is to include more basic information for beginners, so that would be a big help!

Reply:
@leeoniya I think we should just copy the paragraph from your demo, I always enjoy reading it 🙈